services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -117,56 +134,54 @@ async def list_services(
         ]
         return services_overview
 
-    # let's get all the services access rights
-    get_services_access_rights_task = services_repo.list_services_access_rights(
-        key_versions=list(services_in_db.keys()), product_name=x_simcore_products_name
-    )
-
-    # let's get the service owners
-    get_services_owner_emails_task = groups_repository.list_user_emails_from_gids(
-        {s.owner for s in services_in_db.values() if s.owner}
-    )
-
-    # getting services from director
-    get_registry_services_task = director_client.get("/services")
+    # caching this steps brings down the time to generate it at the expense of being sometimes a bit out of date
+    @cached(ttl=DIRECTOR_CACHING_TTL)
+    async def cached_registry_services() -> Deque[Tuple[str, str, Dict[str, Any]]]:
```

Comment:
MINOR: the name should not incorporate ``cached_`` since this is a decorated property (can have it or not)
Q: here you do not worry about the key-name? does this mean that different requests to this entry will reuse this cache? it so, does the access of different request might play a role here?

Reply:
aiocache default key uses module name, method name and so on. here there is no key to worry about. and all users use the same one